targets/python/source/source/.vscode/launch.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Please add .vscode/ to the top level .gitignore for this repo, and un-add all these .vscode files.